tools/simple-test/tester.py

```diff
@@ -0,0 +1,13 @@
+import yaml
+import json
+import sys
+from jsonschema import validate
+
+schema_path = sys.argv[1]
+sample_path = sys.argv[2]
+
+with open(sample_path, 'r') as sample_data:
+    with open(schema_path, 'r') as schema_data:
+        schema_dict = yaml.safe_load(schema_data)
+        sample_dict = json.load(sample_data)
+        validate(instance=sample_dict, schema=schema_dict)
```

Comment:
It’s actually not necessary to enclose the _validate_ call in the _open_ context. The files are already loaded and can be closed by that time.

```suggestion
with open(sample_path, 'r') as sample_data:
    with open(schema_path, 'r') as schema_data:
        schema_dict = yaml.safe_load(schema_data)
        sample_dict = json.load(sample_data)
validate(instance=sample_dict, schema=schema_dict)
```